pkg/kind/kind.go

```diff
@@ -160,11 +161,11 @@ apiVersion: kind.x-k8s.io/v1alpha4
 name: %s
 nodes:
 - role: control-plane
-  image: kindest/node:%s
+  image: %s
```

Comment:
In #239 we're adding the ability to change the version tag on the `kindest/node` image, so we'll eventually need to take that into account. Though maybe we merge this first and fix it there afterwards (@csantanapr thoughts?)